GX-TimeFrequency/src/minimax_tau.F90

```diff
@@ -2915,42 +2914,40 @@ subroutine get_points_weights_tau(grid_size, e_range, ac_we, ierr)
     !> Begin work
     ierr = 0
 
-    if (any(tau_npoints_supported == grid_size)) then
-       ! Find location of grid size
-       kloc = findloc(tau_npoints_supported, grid_size, 1)
-       bup = energy_ranges_grids(kloc)
-
-       ! Allocate and set type elements
-       allocate(aw%energy_range(bup))
-       allocate(aw%aw_erange_matrix(2*grid_size, bup+1))
-       call set_aw_array(grid_size, aw)
+    if (.not. any(tau_npoints_supported == grid_size)) then
+       ierr = 1
+       _REGISTER_EXC("The grid size you chose is not available.")
+       return
+    end if
 
+    ! Find location of grid size
+    kloc = findloc(tau_npoints_supported, grid_size, 1)
+    bup = energy_ranges_grids(kloc)
 
-       ! Select energy region with binary search
-       ien = find_erange(bup, aw%energy_range, e_range)
+    ! Allocate and set type elements
+    allocate(aw%energy_range(bup))
+    allocate(aw%aw_erange_matrix(2*grid_size, bup+1))
+    call set_aw_array(grid_size, aw)
 
-       ! Scale grids for large sizes when erange falls in the
-       ! first energy range
-       e_ratio = 1.0_dp
-       if (ien == 1 .and. grid_size > 20) then
-          e_ratio = aw%energy_range(1)/e_range
-          if (e_ratio > 1.5_dp) then
-             e_ratio = e_ratio/1.5_dp
-          endif
-       end if
+    ! Select energy region with binary search
+    ien = find_erange(bup, aw%energy_range, e_range)
 
-       ac_we(:) = aw%aw_erange_matrix(:, ien)
+    ! Scale grids for large sizes when erange falls in the first energy range
+    e_ratio = 1.0_dp
+    if (ien == 1 .and. grid_size > 20) then
+       e_ratio = aw%energy_range(1)/e_range
+       if (e_ratio > 1.5_dp) then
```

Comment:
Can we get a comment or reference to why 1.5?

Reply:
This, AFAIK is heuristics.

Reply:
Sure, but if this was shown somewhere or tested in some context, a comment is helpful

Reply:
@fdelesma Awesome, thanks for clarifying. @Panadestein could you add this to the documentation?